pkgs/top-level/aliases.nix

```diff
@@ -728,6 +728,7 @@ mapAliases {
   imagemagick7Big = imagemagickBig; # Added 2021-02-22
   imagemagick7 = imagemagick; # Added 2021-02-22
   imagemagick7_light = imagemagick_light; # Added 2021-02-22
+  imgtool = mcuboot-imgtool; # Added 2024-10-08
```

Comment:
Please add this alias to `python-aliases.nix` instead of top-level `aliases.nix`.

Reply:
Done.